src/antsibull/schemas/validators.py

```diff
@@ -48,3 +50,16 @@ def convert_bool(value):
         value = _is_truthy_int(value)
 
     return value
+
+
+def convert_path(value):
```

Comment:
good addition! would have been annoying not to have support for this

Reply:
Especially since the example mentions exactly this :D